### ISSUE TITLE
[New features] may be better use metabase instead of superset? 

### ISSUE THREAD

**Issue by YuriyGavrilov** [rylativity/container-analytics-platform#4]
Superset seams like more complicated at the beginning.

**Comment by rylativity** [comment]
I really like that suggestion, @YuriyGavrilov. I haven't used Metabase, but it does seem like it would be a simpler BI component to integrate given that it's a single container.

I will open a new branch to get it integrated, test it out, and add some usage examples.  Thanks for the callout!

**Comment by YuriyGavrilov** [comment]
Nice 😊 thanks 🙏 
By the way you could also try this if you want instead of Trino. 

docker run -d --rm --name cedrus-server -p 8080:8080 \
  cr.yandex/crpjtvqf29mpabhmrf1s/cedrus:422-2
  
This is same Trino but on steroids in term of execution plan in some query’s . Otherwise just know that it is exist. Here some details 
https://www.cedrusdata.ru/

**Comment by YuriyGavrilov** [comment]
@rylativity I recently discovered the querybook from Pinterest. So it also could be a good solution. https://www.querybook.org.

**Comment by rylativity** [comment]
@YuriyGavrilov Yeah, I opened a branch to start integrating Metabase and realized that it doesn't support Trino. Thinking through how to proceed.

I will take a look at the other tools you mentioned. Thanks for the suggestions!

**Comment by YuriyGavrilov** [comment]
@rylativity great 👍 will keep in touch. thanks for the effort

**Comment by YuriyGavrilov** [comment]
@rylativity you could use https://adoptium.net/temurin/releases/ to avoid this https://github.com/trinodb/trino/issues/25427
